.circleci/config.yml

```diff
@@ -271,6 +274,27 @@ jobs:
     <<: *package_nginx_module_base
     <<: *image_ubuntu_16_04
 
+  # Builds the dynamic plugin as well as the shared lib.
```

Comment:
The CMAKE_ARGS seem to say a different thing from this comment.

Reply:
"I changed my mind" (and now I updated the comment)